host/src/main/java/com/instana/dc/host/AbstractHostDc.java

```diff
@@ -76,11 +95,45 @@ public void initMeters(OpenTelemetry openTelemetry) {
         initMeter(openTelemetry, HostDcUtil.MeterName.FILESYSTEM);
         initMeter(openTelemetry, HostDcUtil.MeterName.PROCESSES);
         initMeter(openTelemetry, HostDcUtil.MeterName.PAGING);
+        initMeter(openTelemetry, HostDcUtil.MeterName.IBMQMGR);
     }
 
+    private void addShutdownHook() {
```

Comment:
Those code should not be in the AbstraceHostDc class because other host DC implementation does not need it.